modules/monitoring-searching-and-filtering-metrics-targets.adoc

```diff
@@ -29,4 +28,4 @@ You can filter the list of targets by status and source. The following filtering
 ** *User*. User targets relate to user-defined projects. These projects are user-created and can be customized. 
 
 You can also use the search box to find a target by target name or label.
-Select *Text* or *Label* from the search box menu to limit your search.
\ No newline at end of file
+Select *Text* or *Label* from the search box menu to limit your search.
```

Comment:
I couldn't find text. if found something similar in each preview.

Except where noted, content looks good and is accurate across the distros.

Reply:
I combined the content from several procedures into a single new procedure (file:///home/hardesty/repos/openshift-docs/_preview/openshift-enterprise/osdocs-3797-monitoring-port/monitoring/managing-metrics.html#getting-detailed-information-about-a-target_managing-metrics). But, I neglected to remove the old procedure. I've deleted it.